src/lib/translators/mongo.ts

```diff
@@ -545,11 +548,22 @@ const mapper: Partial<StepMatcher<MongoStep>> = {
     },
   }),
   filter: filterstepToMatchstep,
-  formula: (step: Readonly<S.FormulaStep>) => ({
-    $addFields: {
-      [step.new_column]: buildMongoFormulaTree(math.parse(step.formula)),
-    },
-  }),
+  formula: (step: Readonly<S.FormulaStep>) => {
+    // replace all `[column name]` values by a pseudo and store it in `pseudos`
+    // `buildMongoFormulaTree` will re-put the name back
+    const pseudos: { [key: string]: string } = {};
```

Comment:
It's a matter of style but I'm getting used to `Record<string, string>` and now I think it's a bit clearer. What do you think?